### ISSUE TITLE
Request to add graphics card processing and hardware acceleration capabilities

### ISSUE THREAD

**Issue by huzesama** [opentrack/opentrack#1608]
Request to add graphics card processing and hardware acceleration capabilities

**Comment by sthalik** [comment]
Are you sure you've got the right project?

**Comment by huzesama** [comment]
Yes, I hope opentrack can use Core Graphics card which is not commonly used to share the pressure of calculation. Actually, I don't know if it's an under-configuration problem, because I've been running into a lot of problems with the Eurotruck Simulator 2, and even stopped working. (The above is translated through the translation software, do not know whether the meaning is wrong)

**Comment by sthalik** [comment]
The neuralnet tracker is the only one that could benefit from it. For the rest that's too much copying from and to VRAM. What do you think, @DaWelter? Could be as simple as enabling selection of the ONNX backend.

**Comment by automaticgiant** [comment]
that would be dope
then we could avoid extra gpu loading with NPU support (ie https://onnxruntime.ai/docs/execution-providers/Vitis-AI-ExecutionProvider.html)

**Comment by DaWelter** [comment]
Yeah, I had something started https://github.com/opentrack/opentrack/pull/2069 and then got side tracked ...
